src/browserenginekit.cs

```diff
@@ -1194,5 +1222,134 @@ public enum BEAccessibilityNotification : long {
 		[Field ("BEAccessibilityValueChangedNotification")]
 		Changed,
 	}
+
+	[iOS (18, 2), TV (18, 2), Mac (15, 2), MacCatalyst (18, 2)]
+	[BaseType (typeof (NSObject))]
+	interface BEAccessibilityTextMarker : INSCopying, INSSecureCoding
```

Comment:
Shouldn’t this be using the `I` less version of the protocol names? Or did we fix that?

```suggestion
	interface BEAccessibilityTextMarker : NSCopying, NSSecureCoding
```